NEWgIT/GitContextFactory.cs

```diff
@@ -0,0 +1,15 @@
+namespace NEWgIT;
+
+public class GitContextFactory : IDesignTimeDbContextFactory<GitContext>
+{
+    public GitContext CreateDbContext(string[] args)
+    {
+        var configuration = new ConfigurationBuilder().AddUserSecrets<Program>().Build();
+        var connectionString = configuration.GetConnectionString("ConnectionString");
+
+        var optionsBuilder = new DbContextOptionsBuilder<GitContext>();
+        // optionsBuilder.UseSqlServer(connectionString);
```

Comment:
Også kommentar.
Er det ment til at fungere som TODO? For så kunne det godt tænkes at vi skal til at kigge på at sætte issues op på vores repo.